src/clients/discord/discord.voice.service.ts

```diff
@@ -97,17 +102,43 @@ export class DiscordVoiceService {
       guildId: channel.guildId,
     });
 
+    this.voiceChannelId = channel.id;
+
     this.jellyfinWebSocketService.initializeAndConnect();
 
     if (this.voiceConnection === undefined) {
       this.voiceConnection = getVoiceConnection(member.guild.id);
     }
+
     this.voiceConnection?.on(VoiceConnectionStatus.Disconnected, () => {
       if (this.voiceConnection !== undefined) {
         this.playbackService.getPlaylistOrDefault().clear();
         this.disconnect();
       }
     });
+
+    const mem = member;
+    setTimeout(() => {
+      // leaving channel cuz no listener left
+      const interaval = setInterval(async () => {
```

Comment:
You never store the interval ID and never clear the interval. Please read the documentation https://developer.mozilla.org/en-US/docs/Web/API/Window/setInterval#return_value
This will lead to memory leaks, random exceptions and other unwanted behavior.